Chinese/Wolf/fight_wolf.md

```diff
@@ -0,0 +1,25 @@
+晚上，一只大灰狼溜进了树林，踩疼了一串喇叭花。喇叭花闻出了大灰狼味，立刻变成了会报警的小喇叭： 
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.